gke/scf-config-values.yaml

```diff
@@ -0,0 +1,28 @@
+### example deployment configuration file
+### scf-config-values.yaml
+
+env:
+  UAA_PORT: 443
+  UAA_PUBLIC_PORT: 443
+  GARDEN_ROOTFS_DRIVER: "overlay-xfs"
+  DEFAULT_STACK: cflinuxfs3
+
+kube:
+  storage_class:
+    persistent: "persistent"
+    shared: "persistent"
+  auth: "rbac"
+
+  registry:
+    hostname: "registry.suse.com"
+    username: ""
+    password: ""
+  organization: "cap"
+
+ingress:
+  enabled: true
+  annotations:
+    nginx.ingress.kubernetes.io/proxy-body-size: 1024m
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    cert-manager.io/acme-challenge-type: dns01
+    cert-manager.io/acme-dns01-provider: gcp-clouddns-provider
```

Comment:
I think lines 27 and 28 are no longer required - can you test please without those?

https://cert-manager.io/docs/usage/ingress/#supported-annotations

Reply:
Worked fine without; removed in a3e60a5.